client/core/core.go

```diff
@@ -531,15 +528,20 @@ func (c *Core) CreateWallet(appPW, walletPW string, form *WalletForm) error {
 		return fmt.Errorf(s, a...)
 	}
 
+	err = wallet.Unlock(walletPW, aYear)
+	if err != nil {
+		return initErr("%s wallet authentication error: %v", unbip(assetID), err)
```

Comment:
Again not a change in this PR, but should `xcWallet` have a `Disconnect` method (for `initErr` to call) so that `xcWallet.hookedUp` can get set to `false`?